### PR TITLE
fix a 4 typos + Dnipro

### PR DESCRIPTION
**Pull request by MkrV**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,20 +15,20 @@ FB - https://www.facebook.com/activecorruptiontracking/
 
 Prerequisites
 
-    Python 3.4
+    Python 3.5
```
Comment (autogestion) [inline README.md]: I tried python 3.5 month ago, and project do not work properly

File: README.md [hunk 2]
```diff
@@ -15,20 +15,20 @@ FB - https://www.facebook.com/activecorruptiontracking/
...
 Create a virtual environment
 
-    virtualenv --python=/usr/bin/python3.4 ctracker
+    virtualenv --python=/usr/bin/python3.5 ctracker
     source ctracker/bin/activate
 
 Install dependencies and project
 
-    sudo apt-get install python3.4-dev libpq-dev postgresql postgresql-contrib postgis postgresql-9.3-postgis-2.1 memcached
+    sudo apt-get install python3.5-dev libpq-dev postgresql postgresql-contrib postgis postgresql-9.5-postgis-2.2 memcached
```
Comment (autogestion) [inline README.md]: also, there was problems with postgresql-9.5-postgis-2.2

File: init_geo_data/AL1_05132016.geojson [hunk 3]
```diff
@@ -6,7 +6,7 @@
...
..., [ 33.46, 48.48 ], [ 33.46, 48.5 ], [ 33.44, 48.5 ], [ 33.45, 48.53 ], [ 33.48, 48.54 ], [ 33.56, 48.56 ], [ 33.58, 48.56 ], [ 33.61, 48.58 ], [ 33.72, 48.62 ], [ 33.77, 48.65 ], [ 33.8, 48.65 ], [ 33.82, 48.67 ], [ 33.77, 48.68 ], [ 33.76, 48.7 ], [ 33.69, 48.7 ], [ 33.66, 48.71 ], [ 33.6, 48.72 ], [ 33.56, 48.76 ], [ 33.56, 48.77 ], [ 33.57, 48.79 ], [ 33.59, 48.79 ], [ 33.62, 48.78 ], [ 33.66, 48.81 ], [ 33.69, 48.8 ], [ 33.69, 48.79 ], [ 33.71, 48.78 ], [ 33.74, 48.81 ], [ 33.76, 48.81 ], [ 33.77, 48.8 ], [ 33.77, 48.78 ], [ 33.8, 48.8 ], [ 33.81, 48.76 ], [ 33.83, 48.76 ], [ 33.92, 48.87 ], [ 34.04, 48.81 ], [ 34.07, 48.81 ], [ 34.11, 48.79 ], [ 34.16, 48.78 ], [ 34.24, 48.75 ], [ 34.29, 48.78 ], [ 34.3, 48.78 ], [ 34.28, 48.8 ], [ 34.28, 48.81 ], [ 34.32, 48.84 ], [ 34.31, 48.87 ], [ 34.27, 48.89 ], [ 34.28, 48.92 ], [ 34.31, 48.97 ], [ 34.37, 48.99 ], [ 34.37, 49.0 ], [ 34.35, 49.0 ], [ 34.34, 49.01 ], [ 34.4, 49.05 ], [ 34.41, 49.08 ], [ 34.43, 49.07 ], [ 34.61, 49.13 ], [ 34.73, 49.15 ], [ 34.78, 49.17 ], [ 34.83, 49.18 ], [ 34.86, 49.17 ], [ 34.87, 49.17 ], [ 34.86, 49.15 ], [ 34.87, 49.14 ], [ 34.89, 49.15 ], [ 34.91, 49.17 ], [ 34.93, 49.18 ], [ 34.98, 49.16 ] ] ] } },
```
Comment (autogestion) [inline init_geo_data/AL1_05132016.geojson]: region was not renamed, only city (:

**Comment by MkrV** [comment]
I took note of comments.

**Comment by autogestion** [comment]
Could you propose changes as 1 commit?

**Pull request closed without merging**